proto/palomachain/paloma/gravity/msgs.proto

```diff
@@ -169,3 +172,23 @@ message EventOutgoingTxId {
     string message = 1;
     string tx_id = 2;
 }
+
+message MsgUpdateParams {
```

Comment:
If this message is to work with our custom ante handler, it will need to embed `metadata` like all other custom messages as well. It will also need to implement the `ValidateBasic` and `GetSigners` interfaces, like all other custom messages.

Reply:
Yes ,`MsgUpdateParams` has the `metadata` field and it implements `ValidateBasic` and `GetSigners`.